llvm/test/Transforms/LoopVectorize/X86/pr141968-instsimplifyfolder.ll

```diff
@@ -0,0 +1,150 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals none --version 5
+; RUN: opt -passes=loop-vectorize -S %s | FileCheck %s
```

Comment:
Can this test be made independent of the X86 target, by providing something like `-force-vector-width=2`?

Reply:
No, unfortunately. I already tried with different vector widths and vector interleaves, but it doesn't crash.